UnitsNet.Tests/CustomCode/SpeedTests.cs

```diff
@@ -73,7 +73,7 @@ public class SpeedTests : SpeedTestsBase
 
         protected override double UsSurveyFeetPerHourInOneMeterPerSecond => 1.1811E4;
 
-        protected override double MachInOneMeterPerSecond => (1.0 / 340.29);
+        protected override double MachInOneMeterPerSecond =>2.9386699579770E-3;
```

Comment:
Verified at https://www.metric-conversions.org/speed/mach-to-meters-per-second.htm